src/components/Footer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import menuItems from '../services/MenuItems';
+import { getPageUrl } from '../Routes';
+import { spacing, fontSizes } from './styledComponents';
+import logo from '../assets/logo/logo.orange.png';
+import iconFacebook from '../assets/icons/icon.facebook.jpg';
+import iconLinkedIn from '../assets/icons/icon.linkedIn.jpg';
+import iconTwitter from '../assets/icons/icon.twitter.jpg';
+
+
+const socialMedias = [
+  {
+    image: { src: iconFacebook, alt: 'Link to Facebook' },
+    link: getPageUrl('HomePage'),
+  },
+  {
+    image: { src: iconLinkedIn, alt: 'Link to LinkedIn' },
+    link: getPageUrl('HomePage'),
+  },
+  {
+    image: { src: iconTwitter, alt: 'Link to Twitter' },
+    link: getPageUrl('HomePage'),
+  },
+];
+
+const Content = styled(Grid)({ flex: '1' });
+
+const SocialMediasContainer = styled(Grid)({});
+
+const Logo = styled('img')({ width: 'fit-content' });
+
+const Icon = styled('img')({ margin: `${spacing(4)} ${spacing(3)} ${spacing(4)} 0px` });
+
+const CopyrightSpan = styled(Typography)({
+  fontFamily: 'Roboto',
```

Comment:
These individual font characteristics (`fontFamily`, `fontSize`, and `lineHeight`) should be standardized into a theme - then we can use the [Typography API](https://material-ui.com/api/typography/) to style them (see the `variant` property). This way similar text is consistent (and changeable) across the whole site.